lib/nylas/v1/contact.rb

```diff
@@ -0,0 +1,20 @@
+module Nylas
+  module V1
```

Comment:
Hmm, I'm not sure I understand why we need this. The new contact model changes will only be in v4.x of the SDK, where we assume that everybody will be using the new version of the API (server-side). Could you explain why we'd need to have references to v1 and v2 of the API in the SDK?

Reply:
Nah we can kill it. I was mostly using it to grapple with a model that is read only and not searchable since some of the other stuff is. I was still a bit fuzzy on whether we wanted to force people to upgrade both the SDK and their API version, but I think it's cool.